contrib/trace_propagators/build.gradle

```diff
@@ -12,6 +12,8 @@ ext.moduleName = "io.opentelemetry.contrib.trace.propagation"
 dependencies {
     api project(':opentelemetry-api')
 
+    testImplementation libraries.jaeger_client
```

Comment:
I see what you did, please split this change in a separate PR. Also nice catch :)

Reply:
Sure thing, I've created #1177.  As soon as that one is merged I'll rebase this PR.